.gitlab-ci.yml

```diff
@@ -18,16 +18,182 @@ lint:
         npm run lint;
         '
 
-test:
+test 1/16:
```

Comment:
@tegefaulkes can we leave this one out. I want to review the CI/CD changes independently of this. Especially parallel test running should be applied after we get out testing rock solid.

Reply:
I will remove this at the end, right now it's useful for testing.